### DISCUSSION TITLE
Why did I find that the orbit of the Chinese space station has not been updated, and now that the Shenzhou 16 has been launched, and there is no update? The previous version is still available.

### DISCUSSION THREAD

**Discussion by 1234f503** [Stellarium/stellarium#3398] [Q&A] [answered]
Why did I find that the orbit of the Chinese space station has not been updated, and now that the Shenzhou 16 has been launched, and there is no update? The previous version is still available.

**Comment by alex-w** [comment, marked as answer]
Please use Satellites plugin to manage artificial satellites

**Reply by 1234f503** [reply]
Satellite plugins are not available either

**Reply by alex-w** [reply]
If you can see Chinese space station, then you have enabled Satellites plugin. Please see Stellarium User Guide to details for manage plugins and details for Satellites plugin.

**Reply by 1234f503** [reply]
Thank you, you also sent me an email to answer.
